### ISSUE TITLE
Docker image starts, index request fails immediately.

### ISSUE THREAD

**Issue by michael-simons** [martin-ueding/geo-activity-playground#200]
Followed the instructions here https://martin-ueding.github.io/geo-activity-playground/getting-started/docker/

```bash
git clone https://github.com/martin-ueding/geo-activity-playground.git\ncd geo-activity-playground
docker build -t geo-activity-playground .
mkdir data
docker run -p 5001:5000 -v `pwd`/data:/data -it geo-activity-playground
```

Image starts, does some stuff

```
dir data 
total 0
88395811 0 drwxr-xr-x   4 msimons  staff  128 24 Nov 07:28 .
88395258 0 drwxr-xr-x  22 msimons  staff  704 24 Nov 07:28 ..
88395843 0 drwxr-xr-x   4 msimons  staff  128 24 Nov 07:28 Cache
88395842 0 drwxr-xr-x   2 msimons  staff   64 24 Nov 07:28 Strava API
```

Request to `/` gets me an internal server error:

```
2024-11-24 06:30:29 geo_activity_playground.webui.app ERROR Exception on / [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/pandas/core/indexes/base.py", line 3805, in get_loc
    return self._engine.get_loc(casted_key)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "index.pyx", line 167, in pandas._libs.index.IndexEngine.get_loc
  File "index.pyx", line 196, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 7081, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 7089, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'kind'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/geo_activity_playground/webui/app.py", line 38, in index
    return render_template("home.html.j2", **entry_controller.render())
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/geo_activity_playground/webui/entry_controller.py", line 19, in render
    kind_scale = make_kind_scale(self._repository.meta, self._config)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/geo_activity_playground/webui/plot_util.py", line 8, in make_kind_scale
    kinds = sorted(meta["kind"].unique())
                   ~~~~^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pandas/core/frame.py", line 4102, in __getitem__
    indexer = self.columns.get_loc(key)
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pandas/core/indexes/base.py", line 3812, in get_loc
    raise KeyError(key) from err
KeyError: 'kind'```

**Comment by martin-ueding** [comment]
Thanks for the detailed report! I've fixed that in version 0.33.3 that I'll release later tonight.

**Comment by beautiful-orca** [comment]
Documentation for docker can be extended by mentionig directory structure like in https://martin-ueding.github.io/geo-activity-playground/getting-started/docker-compose/#creating-directory-structure-and-composeyml
eg. `mkdir -p /docker/geo-activity-playground/playground/Activities`
